app/routers/vip.py

```
@@ -1,23 +1,24 @@
+import inspect
 import logging
 import os
-import inspect
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)